packages/webpack-config-terra/package.json

```diff
@@ -56,7 +56,7 @@
     "@babel/core": "^7.4.4",
     "core-js": "^3.1.3",
     "regenerator-runtime": "^0.13.2",
-    "webpack": "^4.30.0",
-    "webpack-cli": "^3.3.2"
+    "webpack": "^4.30.0 || 5.0.0",
+    "webpack-cli": "^3.3.2 ||4.0.0"
   }
```

Comment:
Should allowing for webpack 5 be its PR?

Reply:
I mean, sure it could be, but it builds on this PR, it's fairly small, and I haven't gotten much feeback on this review yet.